### ISSUE TITLE
Conflict with Theme My Login: call to undefined function login_header

### ISSUE THREAD

**Issue by juellez** [WP-API/api-console#15]
I get the following error:
```
Fatal error: Call to undefined function login_header() in ...wp-content/plugins/OAuth1/theme/oauth1-authorize.php on line 2
```
after I setup my `config.json` file and hit "Auth" in the lower right hand corner of the console screen. First, a small window pops up and then I see this error. The URL at the moment of the error:
```
http://{myurl}/login?action=oauth1_authorize&oauth_callback=http%3A%2F%2F{myurl}%2Fapi%2Fapi-console%2Fland.html&oauth_token={atoken}
```
If I log out of my site and run it, I DO see a login screen, but as soon as I login (in this flow) I get that error. Seeing as how this error is being triggered on a path not "owned" by this plugin, I did some digging ... and confident it has something to do w/my using a custom login screen at the "/login" URL. 

In summary, to recreate, enable "Theme My Login" plugin. Setup "/login" as your login page and URL.
1. Click Auth in the console. A small window pops up.
2. (a) If I'm logged out, it'll give me the login form. I log in. I get the error.
2. (b) If I'm logged in, I get the error.

I confirmed that if I disable the plugin, the authentication succeeds. But I still don't see anything in the viewer (which is likely a different issue). Even if it's beyond your control to "fix" perhaps there's a documentation update that can be made, since several folks use custom login screens. Thanks!
